pkg/reconciler/events/pubsub/pubsub.go

```diff
@@ -124,51 +120,14 @@ func (r *Reconciler) reconcile(ctx context.Context, pubsub *v1alpha1.CloudPubSub
 		return nil
 	}
 
-	ps, err := r.reconcilePullSubscription(ctx, pubsub)
+	_, err := r.PubSubBase.ReconcilePullSubscription(ctx, pubsub, pubsub.Spec.Topic, resourceGroup, true)
```

Comment:
`true /* isPushCompatible */`

Thank you for this extra refactor to reduce duplicated code!